elixir/support/support.ex

```diff
@@ -118,4 +118,36 @@ defmodule AppIdentity.Support do
   def adjust_timestamp(timestamp, diff, :hours) do
     DateTime.add(timestamp, diff * 24 * 60, :second)
   end
+
+  # The following code is adapted from https://hex.pm/packages/secure_random,
```

Comment:
I have ignored this code. I assume it works.

Reply:
Yes. The tests use it extensively.